src/dashboard/api/cms/category/category.ts

```diff
@@ -2,26 +2,36 @@ import { IPageCategory } from "@/types/cms";
 import { axiosPrivate } from "@/utils/axiosPrivate";
 
 export const getAllPageCategories = async () => {
-  const { data } = await axiosPrivate.get("/cms/category/");
-  return data;
+  const { body } = await fetch(`/api/cms/category/`, {
+    method: "GET",
+  });
+  return body;
 };
 
 export const createPageCategory = async () => {
-  const { data } = await axiosPrivate.post("/cms/category/", { type: [] });
-  return data;
+  return await fetch(`/api/cms/category/`, {
```

Comment:
I'm actually curious whether the `await` has any meaning here. Because, in some API methods you use it, in others you don't. 🙂

Reply:
ahh probably not, it might be just an syntactic sugar 😂. But honestly I'm a bit too lazy to open the code and change it

Reply:
don't torture me

Reply:
It's not a tortute, I just missed that you commented it 😄